ci/linux/Dockerfile

```diff
@@ -51,20 +50,23 @@ RUN apt-get install \
   libxss-dev \
   libxt-dev \
   libxv-dev \
-  libxxf86vm-dev
+  libxxf86vm-dev \
+  libwayland-dev \
+  libxkbcommon-dev 
 
 # Build and install CMake
 ARG CMAKE_VERSION=3.14.5
 RUN curl -fLsS "https://github.com/Kitware/CMake/releases/download/v$CMAKE_VERSION/cmake-$CMAKE_VERSION.tar.gz" | tar -f - -xvzC /tmp && \
   cd /tmp/cmake-$CMAKE_VERSION && \
-  ./configure --system-curl && make && make install && \
+  ./configure --system-curl && make -j6 && make install && \
```

Comment:
what's with the -j6 vs -j4 below? a user who wants -j options can export MAKEFLAGS=-j4 before running the script. note that compiling C++ code with debug info can lead to huge memory usage, so using e.g. -j6 by default may cause the build to fail if there are only like e.g. 4GB RAM available on the build environment.

Reply:
This reduces the build time on our CI system, I tested. The Dockerfile is guided to our CI system.

Reply:
As long as it is a real shell it should probably be `-j$(getconf _NPROCESSORS_ONLN)` if you want to go faster but allow people to run it manually without issues. That way the CPU count is limited to what Docker is allocated, which is under the control of the CI system or the user who is running it at home.

Reply:
I can edit and push force as this tonight.

Reply:
For reference, our Docker image is built with 15GB RAM and 4 cores on Cirrus-CI.